islandora_compound_object.module

```diff
@@ -739,12 +739,13 @@ function islandora_compound_object_retrieve_compound_info(AbstractObject $object
     $next_pid = (isset($siblings[$current_key + 1])) ? $siblings[$current_key + 1] : '';
 
     // Check if perms to show link for parent manage.
-    if (islandora_object_manage_access_callback(array(
+    if ($parent && islandora_object_manage_access_callback(array(
       ISLANDORA_METADATA_EDIT, ISLANDORA_MANAGE_PROPERTIES, ISLANDORA_ADD_DS), $parent)) {
       $parent_url = 'islandora/object/' . $parent_pid . '/manage';
     }
     else {
       $parent_url = FALSE;
+      $parent = (object) array('label' => NULL);
```

Comment:
So if there is no $parent->label, maybe it would be a better coding practice to try to assign a temp variable $parent_label = NULL if not present or $parent->label if present. That way you don't have to invent an object with a fake property.

Reply:
@DiegoPino would this suffice in the info array?
`'parent_label' => ($parent && $parent->label) ? $parent->label : NULL,`